rpc/rpc-wrappers.go

```diff
@@ -513,6 +530,16 @@ func (rac RegistrationAuthorityClient) AdministrativelyRevokeCertificate(ctx con
 	return
 }
 
+// DeactivateAuthorization deactivates a currently valid authorization
```

Comment:
Can you update this comment to mention that `DeactivateAuthorization` also deactivates pending authorizations as well as currently valid ones?